avm/res/network/azure-firewall/main.bicep

```diff
@@ -621,3 +621,20 @@ type networkRuleCollectionType = {
     }[]
   }
 }[]?
+
+type hubIPAddressesType = {
+  @description('Optional. List of private IP addresses associated with AzureFirewall.')
+  privateIPAddress: string?
```

Comment:
If this is a list (as per the description) should it then not be a string array?